articles/logic-apps/custom-connector-overview.md

```diff
@@ -61,7 +61,7 @@ In contrast, non-service provider built-in connectors have the following attribu
 
 * Isn't based on the Azure Functions extensibility model.
 
-* Is directly implemented as a job within the Azure Logic Apps runtime, such as Schedule, HTTP, Request, and XML operations.
+* Is directly implemented as a job within the Azure Logic Apps runtime, such as Schedule, HTTP Request, and XML operations.
```

Comment:
Actually, we have separate triggers named **Request** and **HTTP**. There's no **HTTP Request** trigger:

![image](https://user-images.githubusercontent.com/13206671/193431140-ca652504-e114-4ee9-b00f-c47a5d21428b.png)